docs/en_US/Compression/v2_pruning_algo.rst

```diff
@@ -266,6 +282,8 @@ Usage
    pruner = ADMMPruner(model, config_list, trainer, optimizer, criterion, iterations=10, training_epochs=1)
    masked_model, masks = pruner.compress()
 
+For detailed example please refer to :githublink:`examples/model_compress/pruning/v2/admm_pruning_example.py <examples/model_compress/pruning/v2/admm_pruning_example.py>`
+
```

Comment:
i noticed that some pruners do not have corresponding example, why?

Reply:
This PR was once planned to add basic pruning algorithms, but what you said was a good idea, I will add iterative pruning examples later : )